semanticdb/semanticdb3/javacp.md

```diff
@@ -0,0 +1,480 @@
+### Java
+
+In this section, we exhaustively map Java language features onto SemanticDB.
+As a reference, we use the [JVM Specification][jvms] (referred to as "JVMS" in the text below)
+and [Java Language Specification][jls] (referred to as "JLS" in the text below).
+
+<a name="java-symbol"></a>
+#### Symbol
+
+In this section, we describe the Java symbol format.
+
+<table>
+  <tr>
+    <td><b>Symbols</b></td>
+    <td><b>Format</b></td>
+  </tr>
+  <tr>
+    <td>
+      Global symbols
+      <a href="#symbol">↑</a>
+    </td>
+    <td>
+      <ul>
+        <li>
+          For root package <a href="https://www.scala-lang.org/files/archive/spec/2.12/09-top-level-definitions.html#package-references">[20]</a>,
+          its descriptor.
```

Comment:
What's a root package? I don't think we should be using references to SLS in this section.